### ISSUE TITLE
create gig details page

### ISSUE THREAD

**Issue by joesustaric** [livemusiclocator/lml_frontend_client#25]
When you click on a gig it should show something like

![Image](https://github.com/livemusiclocator/lml_frontend_client/assets/199240/d3a88e8c-d5ab-4686-bd29-c2f3efd5c0cd)

**Comment by threefins** [comment]
OK Some notes for kickoff times :)

I'm assuming that the detail pane will slot in somewhere. i will treat it as being full-width on a mobile screen. That means it can appear superimposed on the map on full width if we like, 

Also assuming that the "Electronic, Ambient" is going to be "tags" perhaps a filtered set of tags but for now all tags. We can have the same UI for tags as in other places I guess. (including special LBMF styles at some point)_.

I'll try and do something with the ticket prices etc that makes sense. And take the emoji as instructional :)

**Comment by joesustaric** [comment]
Yep sounds good, all suggestional stuff.. we need something to refine later :)

**Comment by markryall** [comment]
the genres on acts and tags on gigs are separate in the db but there's no reason not to combine them in either the front or back end if it makes filtering/presentation easier

lbmf at this stage is also a tag that is required or the gig will be filtered out on lbmf.lml.live

**Comment by markryall** [comment]
it'd be cool if someone could do a spike on using some api to get estimated travel time

but then it'd perhaps be much easier to work out how to launch some kind of maps application (which would provide travel time estimates and also directions) and then we don't need to pay to use an api

**Comment by threefins** [comment]
the 'launch an app' bit I think is a well trod path. If it is I'll do it as paert of this. 

I think that Nick already did some more stuff on travel time but I will check back in slack

**Comment by threefins** [comment]
First version is pushed now. I won't put this in done until the rest of the functionality I see needed is either done or covered by other cards.

**Comment by threefins** [comment]
Adding what I put into slack before just incase it is helpful.

Firstly, it's very definitely optimised for layouts under about 500 wide. I figure that is most people viewing on their phones. The wider view is not very good presently. I think we need to reconsider some of the "Macro layout" (A term I just learned this week https://web.dev/learn/design/macro-layouts) ,Secondly, a paucity of data fields meant I didn't have much to show there. The richer information that I as a punter might want to see is missing. We can work to add the blurbs into it easy enough.The information I see as being divided into a few different categories:

- "Boring" event information: Time and date, venue directions, licensing, accessibility of venue, age restrictions, ticket costs
- Interesting event info: lineup, Genres, details about the acts, whether it is part of a festival, "Hosts" (cos that is a thing), band artwork, photos, maybe scanned in fliers etc.  


I figure we want to use boring techniques to show the first set of data and fun techniques to show the second type of thing. Some of these could be powered by tags (like accessiblity and genre tags).Another way to put it : Interesting stuff is what makes me want to go to the gig, Boring stuff is the stuff I need to know when I've decided I am interested in attending the gig.
